### PR TITLE
Minor JupyterHub fixes

### PR DESCRIPTION
**Pull request by youngjeong46**
*Issue #, if available:* None

*Description of changes:* Unit test added for jupyterhub

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/addons/jupyterhub/index.ts [hunk 1]
```diff
@@ -176,6 +175,8 @@ export class JupyterHubAddOn extends HelmAddOn {
         setPath(values, "ingress.enabled", enableIngress);
 
         if (enableIngress){
+            const dep = clusterInfo.getScheduledAddOn(AwsLoadBalancerControllerAddOn.name);
```
Comment (shapirov103) [inline lib/addons/jupyterhub/index.ts:178]: is this related to the order of how dependencies are added? I am a bit unclear to why change from dependable to scheduled addon?

File: lib/addons/jupyterhub/index.ts [hunk 2]
```diff
@@ -108,7 +108,6 @@ export class JupyterHubAddOn extends HelmAddOn {
         this.options = this.props as JupyterHubAddOnProps;
     }
 
-    @dependable(AwsLoadBalancerControllerAddOn.name)
     deploy(clusterInfo: ClusterInfo): Promise<Construct> {
```
Comment (shapirov103) [inline lib/addons/jupyterhub/index.ts:111]: Please address the github actions warning: dependable is defined but never used.

**Review by shapirov103 — COMMENTED** [review]
A couple of small comments

File: lib/addons/jupyterhub/index.ts (hunk 1 above)
Comment (youngjeong46) [reply lib/addons/jupyterhub/index.ts:178]: I just realized this change is not necessary. Will remove this. (still need to run dependable decorator for NLB)

File: lib/addons/jupyterhub/index.ts (hunk 2 above)
Comment (youngjeong46) [reply lib/addons/jupyterhub/index.ts:111]: see above.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**